### ISSUE TITLE
[BUG] Layout of standards summary page

### ISSUE THREAD

**Issue by jeff-horton-ho-sas** [UKHomeOffice/engineering-guidance-and-standards#580]
**Describe the bug**
There are issues with the formatting and naviagtion on the summary of standards page
* Sidebar links are not using GOV.UK styles
* Site navigation sections are missing from sidebar (Accessibility: consistent navigation)
* Double horizontal rule at top of page
* No breadcrumbs

**To Reproduce**
1. Go to [Summary of engineering standards’ requirements](https://engineering.homeoffice.gov.uk/standards-requirements/)

**Expected behavior**
* Shows "Engineering guidance and standards" top level navigation in the sidebar.
* Breadcrumbs (showing as subpage of standards?)
* Links use govuk styles
* Only single horizontal rule above main page content

**Screenshots**
<img width="1168" height="565" alt="Screenshot of the top of 'Summary of engineering standards’ requirements' page. The sidebar only has an 'Other formats' section with links in the browser default blue, there are two horizontal rules between the title and the page content" src="https://github.com/user-attachments/assets/991a043b-c12a-4e1a-929b-a47ef19458cb" />
